src/components/note/note.test.tsx

```diff
@@ -0,0 +1,162 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { EditorState } from "draft-js";
+import Note from ".";
+import LinkPreview from "../link-preview";
+import { ActionPopover, ActionPopoverItem } from "../action-popover";
+import { testStyledSystemMargin } from "../../__spec_helper__/__internal__/test-utils";
+
+test("should render with required props", () => {
+  render(
+    <Note
+      createdDate="23 May 2020, 12:08 PM"
+      noteContent={EditorState.createEmpty()}
+    />
+  );
+
+  expect(screen.getByText("23 May 2020, 12:08 PM")).toBeVisible();
+});
+
+test("should render with provided `title` prop", () => {
+  render(
+    <Note
+      createdDate="23 May 2020, 12:08 PM"
+      noteContent={EditorState.createEmpty()}
+      title="Title"
+    />
+  );
+
+  expect(screen.getByRole("banner")).toHaveTextContent("Title");
+});
+
+test("should render with provided `name` prop", () => {
+  render(
+    <Note
+      createdDate="23 May 2020, 12:08 PM"
+      noteContent={EditorState.createEmpty()}
+      name="Carbon"
+    />
+  );
+
+  expect(screen.getByText("Carbon")).toBeVisible();
+});
+
+test("should render tooltip containing status `timeStamp` when status `text` is hovered", async () => {
+  const user = userEvent.setup();
+  render(
+    <Note
+      createdDate="23 May 2020, 12:08 PM"
+      noteContent={EditorState.createEmpty()}
+      status={{
+        text: "Edited",
+        timeStamp: "23 May 2020, 12:10 PM",
+      }}
+    />
+  );
+
+  const statusText = screen.getByText("Edited");
+  expect(statusText).toBeVisible();
+  await user.hover(statusText);
+
+  const tooltip = await screen.findByText("23 May 2020, 12:10 PM");
+  expect(tooltip).toBeVisible();
+});
+
+test("should render LinkPreviews when passed via the `previews` prop as an array", () => {
+  const previews = [
+    <LinkPreview key="1" url="https://example.com" />,
+    <LinkPreview key="2" url="https://example.com" />,
+  ];
+  render(
+    <Note
+      createdDate="23 May 2020, 12:08 PM"
+      noteContent={EditorState.createEmpty()}
+      previews={previews}
+    />
+  );
+
+  const links = screen.getAllByRole("link");
+
+  expect(links).toHaveLength(2);
+  expect(links[0]).toHaveAttribute("href", "https://example.com");
+});
```

Comment:
**suggestion**: since React handles fragments and arrays identically in this case, we could remove this test to simplify the code. 

```suggestion
```

Reply:
For some reason the component handles different preview types differently so both tests are needed for coverage [here](https://github.com/Sage/carbon/blob/e08800c88a352396118a500763048b982017e80e/src/components/note/note.component.tsx#L108)

Reply:
Ah missed we were using child iteration in the code 👍🏼